### ISSUE TITLE
Javadoc missing

### ISSUE THREAD

**Issue by rmuller** [CloudRail/cloudrail-si-java-sdk#11]
I cannot find the JavaDoc for this library. Preferably available via maven central (or other public repository).
The site has "Java DOCS", but no JavaDocs :)

**Comment by FlorianWendel** [comment]
Hi there,

our official documentation at https://documentation.cloudrail.com/java has all the information you could get from a JavaDoc and more. I do understand though that having JavaDoc, which your IDE can display while you code, is more convenient. We'll look into adding that, thanks for the suggestion!

**Comment by rmuller** [comment]
Yes, that's the "Java DOCS". An indeed, the IDE integration is most convenient form developer's perspective. Also note that not everything is clear from the current official documentation (unless i missed it). Example: `public String openAndAwait(String string, String string1);` (`RedirectReceiver`). What is `string1` (in the examples `currentState`)?
